modules/wss4j/src/org/apache/ws/security/saml/SAMLUtil.java

```diff
@@ -234,10 +246,10 @@ public static SAMLKeyInfo getSAMLKeyInfo(SAMLAssertion assertion, Crypto crypto,
 //            confirmMethod = (String) it.next();
 //        }
 //        boolean senderVouches = false;
-//        if (SAMLSubject.CONF_SENDER_VOUCHES.equals(confirmMethod)) {
+//        if (Subject.CONF_SENDER_VOUCHES.equals(confirmMethod)) {
```

Comment:
remove the commented code

Reply:
fixed in 2f696f6746f53ec59ed149ea7753dbcb09f7b044